server/command.go

```diff
@@ -142,21 +150,29 @@ func (p *Plugin) runStartCommand(args *model.CommandArgs, user *model.User, topi
 		}
 		return authErr.Message, authErr.Err
 	}
-
-	client, _, err := p.getActiveClient(user)
-	if err != nil {
-		p.API.LogWarn("Error creating the client", "err", err)
-		return "Error creating the client.", nil
+	var meetingID int = -1
```

Comment:
this whole function this from here to 209 is very messy and useless code think of writing this cleaner